src/room/GroupCallView.tsx

```diff
@@ -104,8 +106,9 @@ export const GroupCallView: FC<Props> = ({
   const [externalError, setExternalError] = useState<ElementCallError | null>(
     null,
   );
-
-  const [muteAllAudio] = useSetting(muteAllAudioSetting);
+  const muteAllAudioControlled = useObservableEagerState(setOutputEnabled$);
+  const [muteAllAudioFromSetting] = useSetting(muteAllAudioSetting);
+  const muteAllAudio = muteAllAudioControlled || muteAllAudioFromSetting;
```

Comment:
Oh gosh, I guess this usage of `setOutputEnabled$` in the GroupCallView shows that, if we turn this into a state observable, that state observable needs to live not on the CallViewModel, but rather something a bit more global than that. Since we don't yet have anything outside the InCallView refactored to use MVVM, I suppose accepting a bit of duplication or putting the `muteAllAudio` observable somewhere in the global/module scope would be fine with me.

Reply:
oh I see.

Reply:
I moved this into a file called `MuteAllAudioModel.ts` With a comment that this might converge into a `GroupCallViewModel.ts`.
If that is what we think is appropritate. otherwise we can move it into a place where it fits once we have such a place.